src/orion/benchmark/assessment/paralleladvantage.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Parallel Advantage Assessment
+=============================
+"""
+
+from collections import defaultdict
+
+from orion.benchmark.assessment.base import BenchmarkAssessment
+from orion.executor.base import executor_factory
+from orion.plotting.base import durations, parallel_advantage, regrets
+
+
+class ParallelAdvantage(BenchmarkAssessment):
```

Comment:
I would be in favor of renaming to `ParallelAssessment `. I think the term advantage here is a bit more confusing than the term assessment would be.